docker/generic/Dockerfile

```diff
@@ -26,4 +26,15 @@ EXPOSE 8090
 # since documentation still refers to using docker without additional EXPOSE step
 EXPOSE 8080
 
+# Remove the nginx user created from deb package which doesn't create the home folder.
+RUN deluser nginx
```

Comment:
Docker suggests [minimizing the number of layers](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers), would it make sense to chain all of these `RUN` commands together?

Reply:
Thanks.  Done